src/scripts/app.jsx

```diff
@@ -41,6 +42,7 @@ class Root extends React.Component {
             <Route exact path="/" component={StoreManageApp} />
             <Route path="/audio/:id" component={Audio} />
             <Route path="/activity" component={Activity} />
+            <Route path="/embed" component={Embed} />
```

Comment:
I think if you move this line down to line 47 (right before the closing div), that should take it out of the "header" scope and also allow it to be publicly accessible.

Reply:
I tried to move to line 47, but the header is still in the scope as they share the same parent. However, I will make a new ticket for removing the header from the embed page and also to make the embeddable audio player publicly accessible. Thanks Jay!